packages/legacy/core/App/screens/ProofRequest.tsx

```diff
@@ -61,7 +61,10 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, proofId }) => {
   const { agent } = useAppAgent()
   const { t } = useTranslation()
   const { assertNetworkConnected } = useNetwork()
+  console.log('ProofRequest AA1')
```

Comment:
Should remove these console logs?

Reply:
Fixed. I also added this back to linting so its caught by our smart robots. 🤖